common-libs/internal-api-clients/src/services/GenericApiService.ts

```diff
@@ -80,7 +80,7 @@ export default class GenericApiService<TApiSpec extends GenericApiSpec> {
         ...(storageRegion && { 'X-DST-REGION': storageRegion }),
       },
       method,
-      ...(params && { body: JSON.stringify(params, null, 2) }),
+      ...(!!params && { body: JSON.stringify(params, null, 2) }),
```

Comment:
Does this change anything?

Reply:
`internal-api-clients` is compiled in strict TS mode, while `wallet-core-sdk` is compiled in loose TS mode.

`typeof params` is any, and in the original version of the code it was causing compilation error in strict mode because object spreads can only be created from the object types (because if `params` is false-ish, we would end up with `...params`).